### PR TITLE
GoShelly

### PR DESCRIPTION
**Pull request by yoarora**
Code for GoShelly server and client. README and instruction set not yet prepared.

### CODE REVIEW

File: Araali-Tools/GoShelly/.env [hunk 1]
```diff
@@ -0,0 +1,10 @@
+MESSAGE=SOMEONE CONNECTED TO GOSHELLY SERVER
+CHANNEL_ID= 
+EMAIL_ID=all@araalinetworks.com
```
Comment (araalinetworks) [inline Araali-Tools/GoShelly/.env]: Leave empty

File: Araali-Tools/GoShelly/.env [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+PORT=443
+SSLCERTGENEMAIL_SERVER=araaligoshellyserver@gmail.com
+SSLCERTGENEMAIL_CLIENT=araaligoshellyclient@gmail.com
```
Comment (araalinetworks) [inline Araali-Tools/GoShelly/.env]: Leave empty

File: doc/source/guide_GoShelly.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+Setup GoShelly Client
+_____________________
+On the VM or K8S cluster, you wish to test the attack response readiness for, install and configure GoShelly Client 
+using the following instructions
+    
+    **For Virtual Machines(VMs)**
+        Clone the repository from GitHub
+            SSH::
+                git clone git@github.com:arorasoham9/goshelly-client.git
```
Comment (araalinetworks) [inline doc/source/guide_GoShelly.rst:57]: Move to araali git

File: Araali-Tools/GoShelly/.env (hunk 2 above)
Comment (yoarora) [reply Araali-Tools/GoShelly/.env]: Already removed. Will commit latest code to Araali's repo.

**Comment by araalinetworks** [comment]
why are we committing Araali-Tools/GoShelly/.DS_Store?

Remove

**Comment by araalinetworks** [comment]
why submodules? Araali-Tools/GoShelly/cobrashelly and client

**Pull request closed without merging**